tests/test_api.py

```diff
@@ -71,7 +105,68 @@ async def test_api_resource_data_not_found(client, mock_get_resource_empty):
 
 async def test_api_resource_data_table_error(client, rmock):
     rmock.get(TABLES_INDEX_PATTERN, payload=[{"parsing_table": "xxx"}])
-    rmock.get(f"{PG_RST_URL}/xxx", status=502, payload={"such": "error"})
+    rmock.get(f"{PG_RST_URL}/xxx?limit=20", status=502, payload={"such": "error"})
     res = await client.get(f"/api/resources/{RESOURCE_ID}/data/")
     assert res.status == 502
-    assert await res.json() == {"such": "error"}
+    assert await res.json() == {'code': 502, 'detail': {'such': 'error'}, 'title': 'Database error'}
+
+
+async def test_api_percent_encoding_arabic(client, rmock):
+    rmock.get(TABLES_INDEX_PATTERN, payload=[{"parsing_table": "xxx"}])
+    rmock.get(
+        f"{PG_RST_URL}/xxx?%D9%85%D9%88%D8%A7%D8%B1%D8%AF=eq.%D9%85%D9%88%D8%A7%D8%B1%D8%AF&limit=20",
+        status=200, payload={"such": "data"}, headers={"Content-Range": "0-10/10"})
+    res = await client.get(
+        f"/api/resources/{RESOURCE_ID}/data/?%D9%85%D9%88%D8%A7%D8%B1%D8%AF__exact=%D9%85%D9%88%D8%A7%D8%B1%D8%AF")
+    assert res.status == 200
+    body = {
+        'data': {"such": "data"},
+        'links': {},
+        'meta': {'page': 1, 'page_size': 20, 'total': 10}
+    }
+    assert await res.json() == body
+
+
+async def test_api_percent_encoding_latin(client, rmock):
+    rmock.get(TABLES_INDEX_PATTERN, payload=[{"parsing_table": "xxx"}])
+    rmock.get(
+        f"{PG_RST_URL}/xxx?c_est_déjà_l_été=eq.BIDULE&limit=20",
+        status=200, payload={"such": "data"}, headers={"Content-Range": "0-10/10"})
+    res = await client.get(f"/api/resources/{RESOURCE_ID}/data/?C\'est déjà l\'été.__exact=BIDULE")
+    assert res.status == 200
+    body = {
+        'data': {"such": "data"},
+        'links': {},
+        'meta': {'page': 1, 'page_size': 20, 'total': 10}
+    }
+    assert await res.json() == body
+
+
+async def test_api_percent_encoding_cyrillic(client, rmock):
+    rmock.get(TABLES_INDEX_PATTERN, payload=[{"parsing_table": "xxx"}])
+    rmock.get(
+        f"{PG_RST_URL}/xxx?компьютер=eq.Компьютер&limit=20",
+        status=200, payload={"such": "data"}, headers={"Content-Range": "0-10/10"})
+    res = await client.get(f"/api/resources/{RESOURCE_ID}/data/?Компьютер__exact=Компьютер")
+    assert res.status == 200
+    body = {
+        'data': {"such": "data"},
+        'links': {},
+        'meta': {'page': 1, 'page_size': 20, 'total': 10}
+    }
+    assert await res.json() == body
+
+
+async def test_api_with_unsupported_args(client, rmock):
```

Comment:
I don't understand this test case

Reply:
It tests that unsupported arguments are just ignored in the sql query.